### PR TITLE
Kasten K10 documentation

### PR DESCRIPTION
**Pull request by ajbergh**
*Issue #, if available:*

*Description of changes:*

Kasten K10 Addon Documentation

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/addons/kasten-k10.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+kasten-io     aggregatedapis-svc-b45d98bb5-w54pr      1/1     Running   0          1m26s
+kasten-io     auth-svc-8549fc9c59-9c9fb               1/1     Running   0          1m26s
+kasten-io     catalog-svc-f64666fdf-5t5tv             2/2     Running   0          1m26s
+```
+
+In the unlikely scenario that pods that are stuck in any other state, please follow the [support](https://docs.kasten.io/latest/operating/support.html#support) documentation to debug further.
+
+### Validate Dashboard Access
+
+By default, the K10 dashboard will not be exposed externally. To establish a connection to it, use the following `kubectl` command to forward a local port to the K10 ingress port:
```
Comment (shapirov103) [inline docs/addons/kasten-k10.md:137]: Does it make sense to integrate with ingress, e.g if aws loadbalancer controller is installed (ALB Ingress) or nginx? How customers will typically use it?

Comment (ajbergh) [reply docs/addons/kasten-k10.md:137]: I'm following our default guidance in our documentation here which is to not expose the GUI by default. You can fully manage Kasten K10 via kubectl and API calls and no exposed GUI is necessarily required. We make it easy/simple to expose K10 via ingress and loadbalancer through options that are documented.

**Review by shapirov103 — COMMENTED** [review]
This looks good! Will merge after we validate the pattern in the patterns repo. I saw a couple of comments from Young on that PR.

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**